src/defaults.js

```diff
@@ -769,6 +769,13 @@ module.exports = {
      * @instance
      */
     enableContinuousRepaint: false,
+
+    /** @summary Reapply cell properties after `getCell`.
+     * @type {boolean}
+     * @default
+     * @instance
+     */
+    reapplyCellProperties: false
```

Comment:
I made this an opt-out. Breaking change, but better this way as it makes renderer faster and anyway we are planning to flip all the -ins into -outs on this `1.1.0` release and document them in "breaking changes" release notes.